projects/js-packages/charts/src/components/line-chart/line-chart.tsx

```diff
@@ -187,6 +188,7 @@ const LineChart: FC< LineChartProps > = ( {
 	className,
 	margin,
 	withTooltips = true,
+	withTooltipCrosshairs = false,
```

Comment:
Looking at an example use case pcO4xN-2zR-p2, the existing use case of another library which will be eventually replaced by this library, uses only vertical crosshair. I am wondering if we should allow this as an object and pass vertical and horizontal separately under one prop?

Reply:
Done in [84d7bd1](https://github.com/Automattic/jetpack/pull/43921/commits/84d7bd1555e9df137c2f7903950d2e196cce1065)